tests_config/utils/compile-contract.js

```diff
@@ -0,0 +1,34 @@
+const solc = require('solc');
+
+function compileContract(filename, content) {
+  const input = {
+    language: 'Solidity',
+    sources: {
+      [filename]: {
+        content
+      }
+    },
+    settings: {
+      outputSelection: {
+        '*': {
+          '*': ['*']
+        }
+      }
+    }
+  };
+  const output = JSON.parse(solc.compile(JSON.stringify(input)));
+  const compiledContracts = output.contracts[filename];
```

Comment:
This script was made only considering the happy path.
Would be nice if someone could add cases where the compiler fails.

Reply:
What would we check if the compiler fails? In that case there would be no output. We could check that the compiler fails in both cases, but I'm not sure if that's very useful.

Reply:
Just added a commit dealing with this.
The test won't pass unless the developer provides a compilable contract for the test.
This way we'll catch things like compiler versions not up to date on the tests that require compiling.